content/inside-rust/compiler-team-new-members-may-2025.md

```diff
@@ -0,0 +1,33 @@
++++
+path = "inside-rust/2025/05/27/compiler-team-new-members"
+title = "Announcing five new members of the compiler team"
+authors = ["davidtwco and wesleywiser"]
+aliases = ["inside-rust/2025/05/30/compiler-team-new-members.html"]
```

Comment:
The `path` variable is the one that counts for the date, not the one in aliases.

Reply:
fixed here: https://github.com/rust-lang/blog.rust-lang.org/pull/1628